packages/layout/src/node/createInstances.js

```diff
@@ -0,0 +1,59 @@
+import { castArray } from '@react-pdf/fns';
```

Comment:
It looks like a new file in the diff here, but this is actually just an edit/rename of `createInstance.js`. Wasn't sure about renaming it but ultimately I thought it felt better with a new name since it now always returns an array of created element instances.